manifest.yml

```diff
@@ -0,0 +1,18 @@
+applications:
+- name: sdx-downstream
+  buildpack: python_buildpack
+  health-check-type: process
+
+  env:
+    CF_DEPLOYMENT: True
+    SDX_STORE_URL: http://sdx-store:5000"
+    SDX_TRANSFORM_CS_URL: http://sdx-transform-cs:5000
+    SDX_TRANSFORM_CORA_URL: http://sdx-transform-cora:5000
+    SDX_SEQUENCE_URL:  http://sdx-sequence:5000
```

Comment:
Should these relate to the CF URLs (http://sdx-store.apps.devtest.onsclofo.uk)?

Reply:
Your right . I thinkI have done the same on several repos- will revisit